doc/api/tls.md

```diff
@@ -876,6 +876,27 @@ socket.on('end', () => {
 });
 ```
 
+When using an instance of `net.Socket`, use `net.Socket` to upgrade an existing socket. Do not wrap the `net.Socket` in a `TLSSocket`. See the example below for usage of upgrading an existing socket:
```

Comment:
Thank you!
Our [doc style guide](https://github.com/nodejs/node/blob/master/doc/STYLE_GUIDE.md) recommends to word-wrap lines at 80 characters)